docs/proposals/003-applications-outside-argocd-namespace.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Allow Application resources to exist in any namespace
+authors:
+  - "@jannfis"
+sponsors:
+  - TBD
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+creation-date: 2021-06-05
+last-updated: 2021-06-05
+---
+
+# Allow Application resources to exist in any namespace
+
+Improve Argo CDs multi-tenancy model to allow Application CRs to be created
+and consumed from different namespaces than the control plane's namespace.
+
+## Open Questions [optional]
+
+* The major open question is, how to name `Application`s in a scenario where
+  the K8s resource's name isn't unique anymore.
```

Comment:
>  K8s resource's name isn't unique anymore.

Is this the resource that the user is deploying from her git repo?

Reply:
The means the name of the `Application` resource, whether created from Git or in another way. Where as before uniqueness was ensured by Kubernetes because no two Applications could share the same name in the same namespace, now possibly an application named `foo` could exist in namespace `ns-foo`, and another application named `foo` could exist in namespace `ns-bar`.